lisp/ess-r-mode.el

```diff
@@ -2296,10 +2296,11 @@ state.")
           (setq text (if (string-match "[\"’]*$" text)
                          (substring text 0 (match-beginning 0))
                        text)))
-        (add-text-properties beg end (list 'ess-r-help-link-text text))
-        (make-text-button beg end
-                          'type 'ess-r-help-link
-                          'help-echo (format "mouse-2, RET: Help on %s" text))))))
+        (when (member text (ess-s-get-help-topics-function ess-local-process-name))
+          (add-text-properties beg end (list 'ess-r-help-link-text text))
```

Comment:
There is no need to query `ess-s-get-help-topics-function` on every text, could be cached on entrance. 

Could you also please remove quotes on this occasion?

Thanks!

Reply:
`ess-s-get-help-topics-function` has a cache already (ess-help-topics-list)

What do you mean remove quotes? Buttonize `lm` instead of `'lm'`?

Reply:
I mean replace `'lm'` with button `lm`. Those quote are unnatural for links.

Reply:
Oh, that makes sense. Done.